news/2018-03-26-osu-coffee-hour-taiko-gamemode-history.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title: "osu! Coffee Hour: osu!taiko Gamemode History"
+date: 2018-03-27 05:00:00 +0000
+---
+
+If there's one thing that sets osu! apart from other communities, it's the amount of different stories to be told. This episode of the osu! Coffee Hour will be the second in our Gamemode History series, where we dive into the unique story behind each gamemode. We're dedicating our second Gamemode History to osu!taiko, the second gamemode to join the osu! ranks nearly 10 years ago.
+
+[![](/wiki/shared/news/banners/coffee-hour.jpg)](http://itsalmo.st/#osutaikogamemodeshowcase)
+
+## This week's theme
+
+What better time for osu!taiko than the day before the Taiko World Cup live drawings? Joining me ([HappyStick](https://osu.ppy.sh/u/HappyStick)) this Friday will be [Raiden](https://osu.ppy.sh/u/Raiden) and [karterfreak](https://osu.ppy.sh/u/karterfreak).
```

Comment:
but it automatically links to it doesn't it?

Reply:
For me these still link to old site. Not sure if it's a preferences thing, but for extra bulletproofness, best to use [HappyStick](https://osu.ppy.sh/users/256802), [Raiden](https://osu.ppy.sh/users/2239480) and [karterfreak](https://osu.ppy.sh/users/1031958).

This way it'll still work even on the offchance one or more of you namechange since it uses the /u link.